### PR TITLE
docs: add sidecar build for release blog and doc updates

### PR DESCRIPTION
**Pull request by KollaAdithya**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/blogs/release-v126.en.md [hunk 1]
```diff
@@ -37,6 +37,47 @@ Copilot v1.26 brings several new features and improvements:
 ## `storage init` for environment addons
 
 ## Sidecar image build
+You can now build Sidecar container images natively from a Dockerfile. Copilot builds and pushes multiple sidecar container images in parallel
+to the same ECR repository of your main container.
+You can modify your workload manifest as below.
```
Comment (iamhopaul123) [inline site/content/blogs/release-v126.en.md]:
```suggestion
You can now use Copilot to build sidecar container images natively from a Dockerfile, and get them pushed to ECR.
```

Let's not mention parallel building/pushing until the PR gets merged.
You can modify your workload manifest as below.

File: site/content/docs/include/sidecar-config.en.md [hunk 2]
```diff
@@ -2,8 +2,41 @@
 <a id="port" href="#port" class="field">`port`</a> <span class="type">Integer</span>  
 Port of the container to expose (optional).
 
-<a id="image" href="#image" class="field">`image`</a> <span class="type">String</span>  
-Image URL for the sidecar container (required).
+<a id="image" href="#image" class="field">`image`</a> <span class="type">String or Map</span>
```
Comment (iamhopaul123) [inline site/content/docs/include/sidecar-config.en.md]: Maybe we can reuse the one for main container if our implementation uses the same struct 🤔

Comment (KollaAdithya) [reply site/content/docs/include/sidecar-config.en.md]: good call 👍

Comment (KollaAdithya) [reply site/content/docs/include/sidecar-config.en.md]: just a small difference these two is `image` field  is `string` for main container and for sidecar it is `String or Map`

Comment (KollaAdithya) [reply site/content/docs/include/sidecar-config.en.md]: Done. used the same docs of main container.

**Pull request closed without merging**